test/tests/ver-1e/comparison_ver-1e.py

```diff
@@ -39,16 +61,22 @@
 gs = gridspec.GridSpec(1, 1)
 ax = fig.add_subplot(gs[0])
 
-tmap_sol = pd.read_csv("./ver-1e_u_vs_x_steadyState.csv")
-tmap_distance = tmap_sol['x']*1e6
+tmap_sol = pd.read_csv("./gold/ver-1e_vector_postproc_line_0548.csv")
+tmap_distance = tmap_sol['x']
+tmap_distance_microns = tmap_distance*1e6
 tmap_conc = tmap_sol['u']
 
-analytical_sol = pd.read_csv("./analytical_u_vs_x_steadyState.csv")
-analytical_distance = analytical_sol['x']*1e6
-analytical_conc = analytical_sol['u']
+x = tmap_distance
+PyC_conc = c0*(1 + (x/l)*((a*D_PyC)/(a*D_PyC + l*D_SiC) - 1 ) )
+SiC_conc = c0*(((a+l-x)/l)*(a*D_PyC)/(a*D_PyC + l*D_SiC) )
+analytical_conc = (x<a)*PyC_conc+(x>=a)*SiC_conc
+
+RMSE = np.sqrt(np.mean((tmap_conc-analytical_conc)**2))
+err_percent = RMSE*100/np.mean(analytical_conc)
+print(err_percent)
```

Comment:
Could you add this value as a printed `R=xxxx` in the figure so that it appears in the documentation?